### ISSUE TITLE
Array of masks with props

### ISSUE THREAD

**Issue by brunokunace** [beholdr/maska#155]
### Describe the bug

When i send an array of tokens in data-maska prop, the same show all of items with commas and not choose which of them.

### Steps to reproduce

You can see in the reproduction link bellow.

### Reproduction link
https://codesandbox.io/s/hardcore-beaver-4hl0x9?file=/src/App.vue

**Comment by jboada** [comment]
> ### Describe the bug
> When i send an array of tokens in data-maska prop, the same show all of items with commas and not choose which of them.

Hi @brunokunace ,

I hope you are well.

I went over the example on the reproduction link. I entered the numbers from 0 thru 9 and the result is 01.23.45,678.901.234.56.78. I did not enter any dot (.) or comma (,) at all, maska applied them.

What would it be the expected result?

Does maska have to decide which mask to apply? If the answer is yes, what would the criteria be for selecting a mask to apply?
